mdata/aggmetric.go

```diff
@@ -66,6 +66,12 @@ func NewAggMetric(store Store, cachePusher cache.CachePusher, key string, retent
 	return &m
 }
 
+func (a *AggMetric) maybeSyncChunkSaveState(ts uint32) {
```

Comment:
In the chunk cache i've had a similar problem with the method `CacheIfHot`. It doesn't always execute it's functions, only if a condition is true, so I've simply put the condition into the name to make it self explanatory